lib/usd/ui/cmds/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# -----------------------------------------------------------------------------
+# sources
+# -----------------------------------------------------------------------------
+target_sources(${PROJECT_NAME}
```

Comment:
`mayaUsd` vs. `${PROJECT_NAME}` in this file?

Reply:
https://github.com/Autodesk/maya-usd/pull/389#discussion_r399602894